example/printcap.c

```diff
@@ -89,6 +89,9 @@ static void pc_init(void *userdata,
 			printf("\tFUSE_CAP_HANDLE_KILLPRIV_V2\n");
 	if(conn->capable & FUSE_CAP_DIRECT_IO_ALLOW_MMAP)
 			printf("\tFUSE_CAP_DIRECT_IO_ALLOW_MMAP\n");
+	/* Disable the receiving and processing of FUSE_INTERRUPT requests */
+	conn->no_interrupt = 1;
```

Comment:
Not needed here, it just exits the session after that anyway.

Reply:
Okay. Yeah, I enable the `no_interrupt` option in example directory by default since they can be the implementation example for programmers. Besides, these example file systems don't use the interruption mechanism in the code. So disable it by default.